File: c#/program5.cs
using System;
using MyArray;

namespace MyArray
{
    class ArrayX
    {            
        public int[] Arr;

        public ArrayX(int size)
        {
            this.Arr = new int[size];
        }

        public void Accept()
        {
            Console.WriteLine("Enter the elements of the array : ");

            for(int iCnt = 0; iCnt < Arr.Length; iCnt++)
            {
                this.Arr[iCnt] = int.Parse(Console.ReadLine());
            }
        }

        public void Display()
        {
            Console.WriteLine("Elements of the array are : ");

            for(int iCnt = 0; iCnt < Arr.Length; iCnt++)
            {
                Console.Write(this.Arr[iCnt]+"\t");
            }
            Console.WriteLine();
        }

        public bool Search(int search)
        {
            bool flag = false;

            for(int i = 0; i < Arr.Length; i++)
            {
                if(this.Arr[i] == search)
                {
                    flag = true;
                    break;
                }
            }
            return flag;
        }        
    }
}

class program5
{
    public static void Main(string[] args)
    {
        int size = 0;
        int value = 0;
        bool bRet = false;

        Console.WriteLine("Enter the size of the array : ");
        size = int.Parse(Console.ReadLine());

        ArrayX obj = new ArrayX(size);
        obj.Accept();
        obj.Display();

        Console.WriteLine("Enter the number to search in array :");
        value = int.Parse(Console.ReadLine());

        bRet = obj.Search(value);
        if(bRet == true)
        {
            Console.WriteLine("Element is present in array :");
        }
        else
        {
            Console.WriteLine("Element is not present in array : ");
        }
    }
}

File: java/program5.java

import java.util.*;

class ArrayX
{
    protected int Arr[];

    public ArrayX(int iSize)
    {
        Arr = new int[iSize];
    }

    protected void Accept()
    {
        Scanner sobj = new Scanner(System.in);
        
        System.out.println("Enter elements of array ");

        for(int iCnt = 0; iCnt < Arr.length; iCnt++)
        {
            Arr[iCnt] = sobj.nextInt();
        }
    }

    protected void Display()
    {
        System.out.println("Elements of array are :");

        for(int iCnt = 0; iCnt < Arr.length; iCnt++)
        {
            System.out.println(Arr[iCnt]+"\t");
        }
        System.out.println();
    }
}

class MyArray extends ArrayX
{
    public MyArray(int iSize)
    {
        super(iSize);
    }

    public boolean CheckNumber(int iSearch)
    {
        boolean bFlag = false;

        for(int iCnt = 0; iCnt < Arr.length; iCnt++)
        {
            if(Arr[iCnt] == iSearch)
            {
                bFlag = true;
                break;
            }
        }
        return bFlag;
    }
}

class program5
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number of elements :");
        int iSize = sobj.nextInt();

        MyArray obj = new MyArray(iSize);
        obj.Accept();
        obj.Display();

        System.out.println("Enter the numbe you want to search :");
        int iValue = sobj.nextInt();

        boolean bRet = obj.CheckNumber(iValue);
        if(bRet == true)
        {
            System.out.println("Number is present :");
        }
        else
        {
            System.out.println("Number is NOT present :");
        }
    }
}